src/amo/components/InstallWarning/index.js

```diff
@@ -134,8 +142,10 @@ const InstallWarning: React.ComponentType<Props> = compose(
   translate(),
   withExperiment({
     id: EXPERIMENT_ID,
-    variantA: VARIANT_INCLUDE_WARNING,
-    variantB: VARIANT_EXCLUDE_WARNING,
+    variants: [
+      { id: VARIANT_INCLUDE_WARNING, percentage: 0.5 },
+      { id: VARIANT_EXCLUDE_WARNING, percentage: 0.5 },
+    ],
```

Comment:
I suppose we should create this `withExperiment` config only once and share it between the two components.